test/with_dummyserver/test_https.py

```diff
@@ -80,6 +90,27 @@ def test_verified(self):
                 error = call[0][1]
                 self.assertEqual(error, InsecurePlatformWarning)
 
+    def test_verified_context(self):
+        ctx = create_urllib3_context(cert_reqs=ssl.CERT_REQUIRED)
+        ctx.load_verify_locations(DEFAULT_CA)
+        https_pool = HTTPSConnectionPool(self.host, self.port,
+                                         ssl_context=ctx)
+
+        conn = https_pool._new_conn()
+        self.assertEqual(conn.__class__, VerifiedHTTPSConnection)
+
+        with mock.patch('warnings.warn') as warn:
```

Comment:
Oh snap, no problem. Didn't mean for this to be a can of worms, just a mild preference of avoiding mock when the alternative is about the same amount of work. :)